docs/en/latest/practices/how-to-access-Apache-APISIX-Prometheus-Metrics-on-k8s.md

```diff
@@ -0,0 +1,170 @@
+---
+title: How to access Apache APISIX Prometheus metrics on Kubernetes
```

Comment:
Would better have a description here, could do this in another PR.

Reply:
ping @Chever-John  PTAL